### PR TITLE
Translate page title

### PR DESCRIPTION
**Pull request by pvuorela**
@jpetrell

### CODE REVIEW

**Comment by pvuorela** [comment]
From first commit: 
    Calligra TextContentsModelImpl.cpp translates this with i18n(), being
    only instance under models subdirectory, but we don't ship calligra
    translations. Since the title is nothing more complicated, let's just
    translate the page number ourselves.
    
    Nesting level is also hard-coded 0 at this point.

File: plugin/TextDocumentToCPage.qml [hunk 1]
```diff
@@ -41,28 +41,14 @@ Page {
...
-                    verticalCenter: parent.verticalCenter
-                }
-                text: model.title
+                x: Theme.horizontalPageMargin
+                width: parent.width - 2*x
+                anchors.verticalCenter: parent.verticalCenter
+                //% "Page %1"
+                text: qsTrId("sailfish_office-la-page_number").arg(model.contentIndex + 1)
                 color: highlighted || isCurrentItem ? Theme.highlightColor : Theme.primaryColor
                 truncationMode: TruncationMode.Fade
```
Comment (jpetrell) [inline plugin/TextDocumentToCPage.qml]: Shouldn't fade. Luckily not much change of that happening with short label and whole display width.

File: plugin/TextDocumentToCPage.qml [hunk 2]
```diff
@@ -41,28 +41,14 @@ Page {
         delegate: BackgroundItem {
             property bool isCurrentItem: model.contentIndex + 1 == page.document.currentIndex
             Label {
-                anchors {
-                    left: parent.left
-                    leftMargin: Theme.horizontalPageMargin + (Theme.paddingLarge * model.level)
```
Comment (jpetrell) [inline plugin/TextDocumentToCPage.qml]: Dropping hierarchy indentation?

File: plugin/TextDocumentToCPage.qml (hunk 1 above)
Comment (pvuorela) [reply plugin/TextDocumentToCPage.qml]: Hm, not sure if I'd call it necessarily wrong here even if we had proper page titles. Anyhow as it now should be always just one word + number I think the simplicity of delegate is better than making it wrap.

**Comment by jpetrell** [comment]
LGTM

File: plugin/TextDocumentToCPage.qml (hunk 1 above)
Comment (jpetrell) [reply plugin/TextDocumentToCPage.qml]: UI labels should never fade, user content can fade as long as there is a way to see it in full.

Fading out UI labels is especially problematic on cases where the differentiating part is at the end, the page number.

Comment (jpetrell) [reply plugin/TextDocumentToCPage.qml]: UI label = translatable system label

Comment (pvuorela) [reply plugin/TextDocumentToCPage.qml]: Might be translatable, but from user point of view there's no much difference if it was coming from the document itself in this case. If we make wrapping/eliding changes, we should then do it for all the document type table of content pages. 

Then again it should be often possible to view the content, if nothing else then clicking on the item and seeing where it takes.

Granted that the page number as last part here is slightly problematic, but don't think in practice it should ever trigger.

**Pull request merged**